src/c-writer.cc

```diff
@@ -2286,6 +2298,53 @@ void CWriter::WriteElemInitializerDecls() {
   }
 }
 
+void CWriter::WriteFuncRefWrapper(const Func* func) {
+  Write("static ", func->decl.sig.result_types, " ", WrapperRef(func->name));
+  Write("(void *instance");
+  if (func->GetNumParams() != 0) {
+    Indent(4);
+    for (Index i = 0; i < func->GetNumParams(); ++i) {
+      Write(", ");
+      if (i != 0 && (i % 8) == 0) {
+        Write(Newline());
+      }
+      Write(func->GetParamType(i), " ", kLocalSymbolPrefix);
```

Comment:
Can we just use `arg_` here instead of `kLocalSymbolPrefix`?

Reply:
`kLocalSymbolPrefix` is `var_`...?

we mean, we guess? we were going for consistency.

Reply:
I was thinking that for these functions `arg_` would make more sense since they are all arguments to the function.   But I guess we use `var_` for them elsewhere so LGTM.